charts/alluxio/docker/init-users/check_port.sh

```diff
@@ -8,6 +8,24 @@ function printUsage() {
     echo -e " PORT1:PORT2:PORT3..."
 }
 
+function check_port() {
+  ports=$1
+  for port in "${ports[@]}"; do
+    echo "Checking if port $port is in use:"
+    echo "> netstat -ntp | grep $port"
```

Comment:
If we have already set `set -x`, do we still need print `echo "> netstat -ntp | grep $port"` ?

Reply:
Currently we are not using 'set -x'(commented).

Reply:
Why not using `set -x`? I'm curious with the reason?